components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/AESDecryptor.java

```diff
@@ -16,13 +16,13 @@
  * under the License.
  */
 
-package org.wso2.carbon.identity.sso.agent.saml;
+package org.wso2.carbon.identity.sso.agent.sdk;
```

Comment:
Rename to io.asgardio.java.saml.sdk?

Reply:
Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e